example/README.md

```diff
@@ -58,6 +58,14 @@ Not only your ActiveGate presents an endpoint, but also your Dynatrace environme
 2. Deploy Configmap and ActiveGate
    ``kubectl apply -f configmap-saas.yaml``
 
+### Chunk Size configuration
+
+Dynatrace plugin uses a default chunk size that may be too small for Kubernetes monitoring, so the plugin configruation has the buffer configuartion below:
+
+```chunk_limit_size 300K```
+
+This means that the plugin will wait until the buffer reaches that size to send to Dynatrace. If logs are generated very infrequently, that may take a while until logs start to appear in Dynatrace. Adjust this size to your needs, but bare in mind that using a too small value may prevent the logs from ever being sent to Dynatrace in case a single line of log is bigger than the chunk size.
```

Comment:
I'm actually not sure this is true. The docs state that `flush_interval` is 60 seconds by default, so I think a flush will happen every 60 seconds no matter what. Our override of the default value came from initial local development where we were trying to get the plugin to export more quickly but should probably have been removed. I'm going to just remove our override. The fluent docs do a good job of explaining buffering/chunk tuning and I don't think we need to duplicate that here.